spacepy/toolbox/__init__.py

```diff
@@ -1114,27 +1128,27 @@ def get_url(url, outfile=None, reporthook=None, cached=False,
     cached : bool (optional)
         Compare modification time of the URL to the modification time
         of ``outfile``; do not retrieve (and return None) unless
-        the URL is newer than the file.
+        the URL is newer than the file. Is set outfile is required.
```

Comment:
If set, not Is set. I would think that the comparison to outfile implies outfile is required, but maybe not....

Reply:
Fixed, I have messed this up on a few occasions and feel like explicit is good.